features/cli/create.feature

```diff
@@ -247,10 +247,10 @@ Feature: creating 'apps' with CLI
     #Check git github url
     Given an 8 character random string of type :dns952 is stored into the :appname3 clipboard
     When I run the :new_app client command with:
-      | code         | git://github.com/openshift/ruby-hello-world |
-      | image_stream | openshift/ruby                              |
-      | name         | <%= cb.appname3 %>                          |
-      | env          | MYSQL_ROOT_PASSWORD=test                    |
+      | code         | git@github.com:openshift/ruby-hello-world.git |
```

Comment:
The line still use git format, which will requires SSH key.

Reply:
@liangxia I've taken that check out since it's doing `git github url` which if we use the git format, then it runs into the issue of not being allowed.  I think we can just remove this section